deliciousbrains/wp-alert-bars
# Allow template files to be overriden

Resolves #20 

**Steps to Test**
1) Modify composer.json in your local copy of the dbi site to point to the composer path repo containing the wp-alert-bars branch.
2) Create a folder \'wp-alert-bars\' in themes/deliciousbrains.
3) Make a copy of the template files \'alert-bar.php\' or \'alert-bar-countdown.php\' in themes/deliciousbrains/themes/deliciousbrains and make some modifications in the files
4) The templates should be picked up from themes/deliciousbrains/themes/deliciousbrains, so that your modifications are visible in the alert bars.
